mkversion.sh

```diff
@@ -39,14 +39,17 @@ fi
 
 # Let's try to derive the version from git..
 if command -v git >/dev/null; then
-    version_from_git="$(git describe --dirty --always | sed -e 's/-/+git/;y/-/./' )"
+    # do not use --dirty until snapcraft deals with this correctly
+    version_from_git="$(git describe --always | sed -e 's/-/+git/;y/-/./' )"
+    echo "version from git: $version_from_git"
```

Comment:
Adding this echo unconditionally will break the version output when we use `--output-only`